source/codegen/metadata/nixnet/functions.py

```diff
@@ -147,6 +147,12 @@
                 'name': 'sessionRef',
                 'type': 'nxSessionRef_t'
             },
+            {
```

Comment:
Would it work if we put this next to `sizeOfValueBuffer` parameter, since that's the param it is replacing? That way there won't much change in signature from user perspective.

Same for other APIs as well.